js/web/lib/wasm/wasm-factory.ts

```diff
@@ -83,7 +102,7 @@ export const initializeWebAssembly = async(): Promise<void> => {
                 [
                   // This require() function is handled by webpack to load file content of the corresponding .worker.js
                   // eslint-disable-next-line @typescript-eslint/no-require-imports
-                  require('./binding/ort-wasm-threaded.worker.js')
+                  require(`./binding/ort-wasm${simdName}-threaded.worker.js`)
```

Comment:
not sure if this works. I think webpack only checks and handle static filename in require(). if it does not work, we need to use if-else to wrap 2 require() function calls

Reply:
It works with my tests, but I'll change it to if-else for a clear expression

Reply:
close this as we will not use multiple .worker.js